src/floorist/floorist.py

```diff
@@ -65,12 +62,17 @@ def main():
                     # Convert any columns with UUID type to string
                     data = data.astype(uuids)
 
-                    wr.s3.to_parquet(data, target,
-                       index=False,
-                       compression='gzip',
-                       dataset=True,
-                       mode='append'
-                    )
+                    if len(data) > 0:
```

Comment:
What's the complexity of this `len` call? Note that this is a second call of `len(data) > 0` within this loop.
Could `not data.empty` be more performant (if is available for `Iterator[DataFrame]`, or, is the same as the `len()` call?

Reply:
I was told earlier that `len` is the standard approach.